include/zephyr/net/dsa_core.h

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
In commit `net: dsa: introduce new DSA framework` some typos in commit message
s/compoenents/components/
s/implementaion/implementation/

Reply:
Thanks, will fix.